### DISCUSSION TITLE
New for trame, want to ask if trame can realize some functions

### DISCUSSION THREAD

**Discussion by dongyangTHU** [Kitware/trame#89] [General]
Im trying to build a frame with trame, and there are some requests I wonder if trame can realize them:
1.  implant a file source,  (just like paraview web visualizer) which can load data source instead of .pvsm file.
2.  "hover points on" function, which is useful in paraview app, to vusialize the point's imformation, ( including location. intensity, etc)
3. when loading a multiple .vtk files, we want trame can play the order, not switch the time line by user

**Comment by jourdain** [comment]
1. Yes: see [Visualizer based on trame](https://github.com/Kitware/paraview-visualizer)
2. Yes: [that example](https://github.com/Kitware/trame/blob/master/examples/06_vtk/01_SimpleCone/RemoteRendering.py) trigger mouse events that can be processed on the server side 
3. Yes: Just schedule a task for switching files on the server side
